### PR TITLE
Image & Input Components

### PR DESCRIPTION
**Pull request by horsemaker**
~ Added Images Component
~ Added Inputs Component
~ Added Status Badges for Avatars

### CODE REVIEW

File: components/badges/badge.css [hunk 1]
```diff
@@ -53,27 +53,79 @@
...
+	right: 4%;
+	bottom: 12.5%;
+	border: 2.5px solid white;
+}
+
+.badge-avatar .avatar-md ~ .badge-content {
+	right: 4%;
+	bottom: 12.5%;
+	border: 2px solid white;
+} */
```
Comment (amarphule) [inline components/badges/badge.css]: remove comments

File: components/cards/index.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
 		<link rel="stylesheet" href="../../default.css" />
 		<link rel="stylesheet" href="../../main.css" />
 		<link rel="stylesheet" href="style.css" />
-		<link rel="stylesheet" href="card.css" />
+		<!-- <link rel="stylesheet" href="card.css" /> -->
```
Comment (amarphule) [inline components/cards/index.html]: please remove unused commented code

**Review by amarphule — COMMENTED** [review]
Nice work @horsemaker , one suggestion remove unused code(commented)

File: components/badges/badge.css (hunk 1 above)
Comment (horsemaker) [reply components/badges/badge.css]: are thanks! almost forgot 😁

File: components/cards/index.html (hunk 2 above)
Comment (horsemaker) [reply components/cards/index.html]: done 👍

**Review by msdeshmukh009 — APPROVED** [review]
Everything is looking good. I think you should move ahead.

**Review by vijaytembugade — APPROVED** [review]
Wow. I like it brother. Amazing interface.

**Review by poornima-alugubelly — APPROVED** [review]
Good to go!

**Pull request merged**